cmd/osbuild-composer/main.go

```diff
@@ -66,6 +82,9 @@ func main() {
 				KeyTab    string `toml:"keytab"`
 			} `toml:"kerberos,omitempty"`
 		} `toml:"koji"`
+		RemoteWorker *struct {
+			DomainAllowlist []string `toml:"domain_allowlist"`
+		} `toml:"remote_worker,omitempty"`
```

Comment:
Name it `worker` for simplicity. The non-remote case is basically legacy and should be going away (if we need config for that we could always call that `local_worker`).

Reply:
Done!